### DISCUSSION TITLE
Add 'remove feed item' option

### DISCUSSION THREAD

**Discussion by M7amma** [jocmp/capyreader#402] [Feature Requests]
Thanks for adding the swipe function. Please add 'remove feed item' as a swipe action also. The option will delete the feed item from the list.

**Comment by Coool** [comment]
Has this feature been implemented? It would be great if we could also select this function (remove article) from the menu.

Releated:
- #433
- #400
- #310
- #273
- #242
- #240
- #238
- #237
- #233
- #192
- #191
- #185

**Comment by jocmp** [comment]
It hasn't been implemented. I don't have any plans to support deleting articles beyond the "auto delete" feature linked above.

**Comment by Coool** [comment]
@jocmp, it's your final decision? Don't bother you with this question again?

**Reply by jocmp** [reply]
Hey @Coool, yeah, still no plans for this. Please ask about any other features in the future, but I'd say it's final for now.
